assertk/src/jvmMain/kotlin/assertk/assertions/file.kt

```diff
@@ -93,7 +93,7 @@ fun Assert<File>.hasName(expected: String) {
  * Asserts the file has the expected path.
  */
 fun Assert<File>.hasPath(expected: String) {
-    path().isEqualTo(expected)
+    path().isEqualTo(File(expected).path)
```

Comment:
This same change should probably also be applied to `hasParent`.

Reply:
Makes sense, can you go ahead and change that one as well? Would also recommend adding/updating the test so it would fail with the old way.